tests/tests.js

```diff
@@ -1,45 +1,193 @@
-const { describe, it } = require('mocha')
 const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
 
-describe('calculateProductPrice', () => {
-  it('returns the price for a voluntary life product for a single employee', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee'],
-      coverageLevel: [{ role: 'ee', coverage: 125000 }],
-    }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
+const chai = require('chai')
+const sinon = require('sinon')
+const sinonChai = require('sinon-chai')
+const { beforeEach, afterEach, describe, it } = require('mocha')
 
-    expect(price).to.equal(39.37)
+const { expect } = chai
+chai.use(sinonChai)
+
+describe('Pricing', () => {
+  let sandbox,
+    formatPriceSpy,
+    getEmployerContributionSpy,
+    calculateVolLifePricePerRoleSpy,
+    calculateVolLifePriceSpy,
+    calculateLTDPriceSpy
+
+
+  beforeEach(() => {
+    sandbox = sinon.createSandbox()
+
+    formatPriceSpy = sandbox.spy(pricing, 'formatPrice')
+    getEmployerContributionSpy = sandbox.spy(pricing, 'getEmployerContribution')
+    calculateVolLifePricePerRoleSpy = sandbox.spy(pricing, 'calculateVolLifePricePerRole')
+    calculateVolLifePriceSpy = sandbox.spy(pricing, 'calculateVolLifePrice')
+    calculateLTDPriceSpy = sandbox.spy(pricing, 'calculateLTDPrice')
+  })
+
+  afterEach(() => {
+    sandbox.restore()
+  })
+
+  describe('calculateLTDPrice', () => {
+    it('returns price for LTD plan for a single employee without employer contribution', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [
+          { role: 'ee', coverage: 125000 }
+        ],
+      }
+      const price = pricing.calculateLTDPrice(products.longTermDisability, employee, selectedOptions)
+
+      expect(price).to.equal(32.04)
+
+      expect(calculateLTDPriceSpy).to.have.callCount(1)
+    })
+
+  })
+
+  describe('calculateVolLifePrice', () => {
+    it('returns the price for a plan for single employee', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee'],
+        coverageLevel: [
+          { role: 'ee', coverage: 125000 }
+        ],
+      }
+      const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+
+      expect(price).to.equal(43.75)
+
+      expect(calculateVolLifePricePerRoleSpy).to.have.callCount(1)
+    })
+
+    it('returns the price for a plan for an employee with a spouse', () => {
+      const selectedOptions = {
+        familyMembersToCover: ['ee', 'sp'],
+        coverageLevel: [
+          { role: 'ee', coverage: 200000 },
+          { role: 'sp', coverage: 75000 },
+        ],
+      }
+      const price = pricing.calculateVolLifePrice(products.voluntaryLife, selectedOptions)
+
+      expect(price).to.equal(79)
+
+      expect(calculateVolLifePricePerRoleSpy).to.have.callCount(2)
```

Comment:
Nice. This one is different. Here you are saying that if you call `calculateVolLifePrice` with an employee and spouse that **it** calls `calculateVolLifePricePerRoleSpy` twice. That's a great thing to check since it needs to do that to work properly.